### PR TITLE
Add Vehicle, Objects, & Motion Control

### PR DESCRIPTION
**Pull request by Emma-Miller04**

### CODE REVIEW

File: Assets/Course Library.meta
Comment (dantogno) [inline Assets/Course Library.meta:1]: Are all of the changes for 1.1 and 1.2 here, or is this just one of the steps in the tutorial?

**Review by dantogno — COMMENTED** [review]
Are all of the changes for 1.1 and 1.2 here, or is this just one of the steps in the tutorial?

File: Assets/Course Library.meta
Comment (Emma-Miller04) [reply Assets/Course Library.meta:1]: yes all the changes for 1.1 and 1.2 are there.  sorry for the mountain of files.

File: Assets/Scripts/PlayerController.cs [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+public class PlayerController : MonoBehaviour
+{
+    // Start is called before the first frame update
+    void Start()
+    {
+        
+    }
+
+    // Update is called once per frame
+    void Update()
```
Comment (dantogno) [inline Assets/Scripts/PlayerController.cs:14]: I guess you skipped the comment 😜

**Review by dantogno — APPROVED** [review]
Looks good, good work! Next time please make commits for each step of the tutorials.

**Review by TheSkylee — COMMENTED** [review]
everything looks good, just make sure to remember to commit multiple steps

**Pull request merged**